modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/ExperimentHelper.java

```diff
@@ -81,6 +81,8 @@ public static Experiment makeExperiment(
         experimentObject.setUserCap(experimentPojo.getUserCap());
         experimentObject.setCreatorID(experimentPojo.getCreatorId());
         experimentObject.setRuleJson(convertRuleToJson(experimentPojo.getRule()));
+        experimentObject.setHypothesisIsCorrect(experimentPojo.getHypothesisIsCorrect());
+        experimentObject.setResults(experimentPojo.getResults());
```

Comment:
I guess the one question I would have is just to make sure that whatever this technology you guys use to put things in an object and then eventually return JSON handles things like quotes and curly braces, basically the things that are special to JSON?  I assume you must, but if that is a special step you need to take, you'd want to do it here.  Because what if my results consist of:  Our so-called "experiment", which tests curly braces ({ and }) isn't that conclusive.

😄